Frontend/CO.CDP.OrganisationApp/Logging/ErrorCodes.cs

```diff
@@ -0,0 +1,6 @@
+namespace CO.CDP.OrganisationApp.Logging;
+
+public class ErrorCodes
+{
+    public const int CompaniesHouseApiError = 1000;
```

Comment:
This needs to be something easily identifiable. `1000` can easily show up in third party log entries of any random library we use.

Imo, error codes should be strings that share common prefix, i.e. `SIRSI1000` or `CDP1000`. We need to decide on the prefix.

ping @mareg

Reply:
Agreed.